.travis.sh

```diff
@@ -24,6 +24,7 @@ function configure()
       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-Debug} \
       -DGTSAM_BUILD_TESTS=${GTSAM_BUILD_TESTS:-OFF} \
       -DGTSAM_BUILD_UNSTABLE=${GTSAM_BUILD_UNSTABLE:-ON} \
+      -DGTSAM_USE_QUATERNIONS=${GTSAM_BUILD_UNSTABLE:-OFF} \
       -DGTSAM_BUILD_EXAMPLES_ALWAYS=${GTSAM_BUILD_EXAMPLES_ALWAYS:-ON} \
```

Comment:
Is it intentional to use quaternions only when building the unstable library? (Just asking...)

Reply:
I think by default GTSAM uses the matrix-based implementation... probably need to ask frank...

Reply:
Oops, no, this is a typo!! Should be: `GTSAM_USE_QUATERNIONS`, see travis.yml.
BTW I don't remember what the `:-ON` syntax is about. Don't remember doing that...

Reply:
PR on the way

Reply:
> BTW I don't remember what the :-ON syntax is about. Don't remember doing that...

It was me.... a bash thing. That file specifically invokes `bash` in its first line, so there are no compatibility issues to worry about. 

It means: take the value of this variable, or if it's not defined, take this value as default. It's handy to shorten the list of env. vars. passed from the build matrix (we only need to change what is not to be left to defaults), and also easier to read since we don't need to go one by one through the list of variables and assign default values with `if [ ] ...`, etc.

Reply:
OK, cool :-)